packages/esm-patient-search-app/src/compact-patient-search/compact-patient-banner.component.tsx

```diff
@@ -98,9 +98,8 @@ const PatientSearchResults = React.forwardRef<HTMLDivElement, PatientSearchResul
                   }`}</h2>
                   <ExtensionSlot
                     extensionSlotName="patient-banner-tags-slot"
-                    state={{ patient }}
+                    state={{ patient, patientUuid: patient.id }}
```

Comment:
@jwnasambu  is passing the `patientUuid` here since it's needed to display the active  [visit tag](https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/packages/esm-patient-banner-app/src/banner-tags/active-visit-tag.component.tsx#L15)